net-misc/urlscan/urlscan-0.9.1.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6,7} )
+
+inherit distutils-r1
+
+DESCRIPTION="Permit to launch URLs to webrowser inside Mutt mail client"
+HOMEPAGE="https://github.com/firecat53/urlscan"
+SRC_URI="https://github.com/firecat53/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+
+DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
```

Comment:
with EAPI-7 setuptools is a BDEPEND, and provided by distutils-r1.eclass. If you need it in RDEPEND, add it there without inheriting DEPEND :)